spinalcordtoolbox/resample/nipy_resample.py

```diff
@@ -0,0 +1,172 @@
+#########################################################################################
+#
+# Resample data using nipy.
+#
+# ---------------------------------------------------------------------------------------
+# Copyright (c) 2014 Polytechnique Montreal <www.neuro.polymtl.ca>
+# Authors: Julien Cohen-Adad, Sara Dupont
+#
+# About the license: see the file LICENSE.TXT
+#########################################################################################
+import copy
+
+import nipy
+import numpy as np
+from nipy.algorithms.registration.resample import resample as n_resample
+
+import sct_utils as sct
+from msct_image import Image
```

Comment:
I don't think you are using `Image` or `Parser` anywhere in the code.

Reply:
Fixed.